scripts/gen-cli-data.py

```diff
@@ -3,31 +3,48 @@
 import json
 
 if len(sys.argv) < 3:
-    print("You must supply a Jaeger version as the first argument and the output path as the second")
+    print("You must supply a Jaeger version as the first argument and "
+          "the output path as the second")
     sys.exit(1)
 
-jaeger_ver=sys.argv[1]
-output_path=sys.argv[2]
+jaeger_ver = sys.argv[1]
+output_path = sys.argv[2]
 
-with open("%s/data/cli/%s/config.json" % (output_path, jaeger_ver), 'r') as f:
-    cfg=json.load(f)
+with open(f"{output_path}/data/cli/{jaeger_ver}/config.json", 'r') as f:
+    cfg = json.load(f)
 
-def generate(tool, storage='', sampling=''):
-    print('Generating YAML for {}, storage {} and sampling {}'.format(tool, storage, sampling))
 
-    volume='%s/data/cli/%s:/data' % (output_path, jaeger_ver)
-    docker_img='all-in-one' if tool == 'jaeger-all-in-one' else tool
-    docker_image="jaegertracing/%s:%s" % (docker_img, jaeger_ver)
-    storageOrSampling=storage or sampling
+# generate generates the CLI documentation for a given "tool" (e.g.
+# jaeger-collector, jaeger-query, etc.) and feature (e.g. storage=elasticsearch,
+# sampling=adaptive, metrics-storage=prometheus).
+def generate(tool, **kwargs):
```

Comment:
I think using `kwargs` would be simpler and cleaner than adding another "optional" parameter to support metrics storage, especially avoiding the need to extend the `storageOrSampling` variable.